PyInstaller/utils/hooks/subproc/django_import_finder.py

```diff
@@ -100,7 +100,7 @@ def find_url_callbacks(urls_module):
     hiddenimports.append(app_ctx_proc_module)
 
 
-from django.core.urlresolvers import RegexURLPattern, RegexURLResolver
+from django.urls import URLPattern, URLResolver
```

Comment:
Please try to keep this backward-compatible. Maybe like this:
```
try:
    # django 2.x
    from django.urls import URLPattern, URLResolver
except ImportError:
    # django 1.x
    from django.core.urlresolvers import RegexURLPattern as URLPattern, \
             RegexURLResolver as URLResolver
```
In a vew year we can then remove the django 1.0 part.